code/modules/projectiles/guns/energy/nuclear.dm

```diff
@@ -32,6 +32,7 @@
 	force = 8 //looks heavier than a pistol
 	self_recharge = 1
 	modifystate = null
+	var/reliability = 95

```

Comment:
Added a flat out base reliability of 95 since the global reliability system has been removed.
There are ways of reducing the reliability, but the default gun should be quite reliable since its one of the default ERT weapons.